discord_bot.js

```diff
@@ -1,49 +1,46 @@
-var fs = require('fs');
-var _ = require('lodash');
+const fs = require('fs');
+const _ = require('lodash');
 
-process.on('unhandledRejection', (reason) => {
-  console.error(reason);
-  process.exit(1);
-});
+// not needed if you don't have errors
+//-------------------------------------------------
+//process.on('unhandledRejection', (reason) => {
+//  console.error(reason);
+//  process.exit(1);
+//});
 
-try {
-	var Discord = require("discord.js");
-} catch (e){
-	console.log(e.stack);
-	console.log(process.version);
-	console.log("Please run npm install and ensure it passes with no errors!"); // if there is an error, tell to install dependencies.
-	process.exit();
-}
+// you don't have to try to require discord.js. 
+// es6
+const Discord = require("discord.js");
```

Comment:
We added this because people will try to run the bot without doing `npm install` first. This exists purely to provide those people with a friendlier error.

Reply:
Yes. I see what you mean.